README.md

```diff
@@ -0,0 +1,239 @@
+|        OPAC_USE_SCIENCEOPEN  	|         False  	|         True/False            	|       21/11/2021           	|       ativa/desativa a integração de métricas com o ScienceOpen. Se sim, definir como 'True'  |
+|        OPAC_USE_SCITE  	|         False  	|         True/False            	|       21/11/2021           	|       ativa/desativa a integração de métricas com _SCITE. Se sim, definir como 'True'   |
+|        OPAC_SCITE_URL  	|         https://cdn.scite.ai/badge/scite-badge-latest.min.js  	|         https://cdn.scite.ai/badge/scite-badge-latest.min.js            	|       21/11/2021           	|       URL para o SCITE_ JS  |
+|        OPAC_SCITE_METRICS_URL  	|         https://scite.ai/reports/  	|         https://scite.ai/reports/            	|       21/11/2021           	|       URL para o Scite_   |
+|        LOCAL_ZONE  	|         'America/Sao_Paulo'  	|         'America/Sao_Paulo'            	|       21/11/2021           	|       Localização para data   |
+|        OPAC_USE_SENTRY  	|         False  	|         True/False            	|       21/11/2021           	|       ativa/desativa a integarção com Sentry, se sim definir como: 'True'   |
+|        OPAC_SENTRY_DSN  	|         None  	|         DSN do Sentry            	|       21/11/2021           	|       DSN definido pelo sentry para este projeto. Utilizado só se OPAC_USE_SENTRY == True   |
+|        OPAC_BUILD_DATE  	|         None  	|         05/11/2002, 09/02/2009            	|       21/11/2021           	|       data de build. definida em tempo de construção da imagem   |
+|        OPAC_VCS_REF  	|         None  	|         None            	|       21/11/2021           	|       commit do código. definida pelo travis em tempo de construção da imagem.   |
+|        OPAC_WEBAPP_VERSION  	|         None  	|         3.44.5,3.43.2            	|       21/11/2021           	|       'versão do OPAC WEBAPP'. definida pelo travis em tempo de construção da imagem. definida pelo travis em tempo de construção da imagem.   |
+|        OPAC_WTF_CSRF_ENABLED  	|         True  	|         True/False            	|       21/11/2021           	|        ativa/desativa o recurso de CSRF   |
+|        OPAC_WTF_CSRF_SECRET_KEY  	|         JGvNWiwBIq2Iig89LWbV  	|                     	|       21/11/2021           	|        chave para segurança nos formulários WTF.   |
+|        READCUBE_ENABLED  	|         False  	|          True/False           	|       21/11/2021           	|        ativa/desativa a exibição do link para o ReadCube, se sim definir como: 'True'    |
+|
+|        OPAC_SSM_SCHEME  	|         https  	|          http/https           	|       21/11/2021           	|        Protocolo de conexão com SSM. Opções: 'http' ou 'https' - (default: 'https')    |
+|        OPAC_SSM_DOMAIN  	|         ssm.scielo.org  	|          URL: ssm.scielo.org dam.doamin.suffix           	|       21/11/2021           	|        Dominio/FQDN do conexão com SSM. Ex: 'homolog.ssm.scielo.org    |
```

Comment:
Trocar `do conexão` por `da conexão`
Verificar se a forma de apresentar os valores possíveis está adequada: `URL: ssm.scielo.org dam.doamin.suffix`, quero dizer, remover o termo `URL:` e colocar uma vírgula entre os domínios (se for o caso).